server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -546,6 +555,11 @@ export class ScribeLambda implements IPartitionLambda {
 		}
 	}
 
+	private markDocumentAsCorrupt(message: IQueuedMessage) {
+		this.isDocumentCorrupt = true;
+		this.prepareCheckpoint(message, CheckpointReason.MarkAsCorrupt, true);
```

Comment:
use 3rd parameter of `this.isDocumentCorrupt`? Or just use the `this.isDocumentCorrupt` inside the `prepareCheckpoint` directlly?